### PR TITLE
Adding watcher documentation and parameters

### PR DESCRIPTION
**Pull request by evyatarmeged**
### Description
Adding watcher documentation and parameters

### CODE REVIEW

File: docs/reference/configuration/network-mapper/README.mdx [hunk 1]
```diff
@@ -47,6 +49,12 @@ DNS is a common network protocol used for service discovery. When a pod (`checko
 ### Active TCP connections
 DNS responses will only appear when new connections are opened. To handle long-lived connections, the network mapper also queries open TCP connections in a manner similar to `netstat` or `ss`. The IP addresses are used for the [service identity resolving process](/reference/service-identities), as above.
 
+### Kafka logs
+TBD
+
+### Istio sidecar metrics
+TBD
+
```
Comment (evyatarmeged) [inline docs/reference/configuration/network-mapper/README.mdx:57]: Should I add these sections ?

File: docs/reference/configuration/network-mapper/README.mdx [hunk 2]
```diff
@@ -47,6 +49,13 @@ DNS is a common network protocol used for service discovery. When a pod (`checko
 ### Active TCP connections
 DNS responses will only appear when new connections are opened. To handle long-lived connections, the network mapper also queries open TCP connections in a manner similar to `netstat` or `ss`. The IP addresses are used for the [service identity resolving process](/reference/service-identities), as above.
 
+### Kafka logs
+the Kafka watcher periodically examines logs of kafka servers provided by the user, parses them and deduces topic-level access to Kafka from pods in the cluster.
```
Comment (orishoshan) [inline docs/reference/configuration/network-mapper/README.mdx:53]:
```suggestion
The Kafka watcher periodically examines logs of Kafka servers provided by the user through configuration, parses them and deduces topic-level access to Kafka from pods in the cluster.
```

File: docs/reference/configuration/network-mapper/README.mdx [hunk 3]
```diff
@@ -37,8 +37,10 @@ recommendationservice in namespace otterize-ecom-demo calls:
 ## How does the network mapper work?
 
 ### Components
+- Mapper - the mapper is deployed once, and resolves service names using the Kubernetes API with traffic information reported by the sniffers and watchers.
 - Sniffer - the sniffer is deployed to each node using a DaemonSet, and is responsible for capturing node-local DNS traffic and inspecting open connections.
-- Mapper - the mapper is deployed once, and resolves service names using the Kubernetes API with traffic information reported by the sniffers.
+- Kafka watcher (**experimental**) - the Kafka watcher is deployed once (when enabled) and is responsible for capturing Kafka topic level access connections.
```
Comment (orishoshan) [inline docs/reference/configuration/network-mapper/README.mdx:42]:
```suggestion
- Kafka watcher (**experimental**) - the Kafka watcher is deployed once per cluster and is responsible for detecting accesses to Kafka topics, which services perform those accesses and which operations they use.
```

File: docs/reference/configuration/network-mapper/README.mdx [hunk 4]
```diff
@@ -37,8 +37,10 @@ recommendationservice in namespace otterize-ecom-demo calls:
...
+- Istio watcher (**experimental**) - the Istio watcher is deployed once (when enabled) and is responsible for capturing HTTP specific connections that are happening within the Istio service mesh.
```
Comment (orishoshan) [inline docs/reference/configuration/network-mapper/README.mdx:43]:
```suggestion
- Istio watcher (**experimental**) - the Istio watcher is deployed once per cluster and queries Istio Envoy sidecars for HTTP traffic statistics, which are used to detect HTTP traffic with paths.
```

File: docs/reference/configuration/network-mapper/README.mdx [hunk 5]
```diff
@@ -37,8 +37,10 @@ recommendationservice in namespace otterize-ecom-demo calls:
 ## How does the network mapper work?
 
 ### Components
+- Mapper - the mapper is deployed once, and resolves service names using the Kubernetes API with traffic information reported by the sniffers and watchers.
```
Comment (orishoshan) [inline docs/reference/configuration/network-mapper/README.mdx:40]:
```suggestion
- Mapper - the mapper is deployed once per cluster, and receives traffic information from the sniffer and watchers, and resolves the information to communications between (service identities)[/service-identities].
```

**Review by orishoshan — APPROVED** [review]

**Pull request merged**